README.md

```diff
@@ -29,7 +39,29 @@ gulp.task('example', function() {
 });
 ```
 
-### decompress()
+### Custom compression
+
+Files are processed with maximum compression and output files have the `.brotli` extension.
+
+```javascript
+var gulp   = require('gulp');
+var brotli = require('gulp-brotli');
+
+gulp.task('example', function() {
+  return gulp.src('path/to/input')
+    .pipe(brotli.compress({
+      extension: 'brotli',
+      brotliParams: {
```

Comment:
You're moving `brotliParams` inside another object, that's a breaking change so I'd rather the `extension` property was added to the `brotliParams` object instead.